src/scripts/modules/media/body/body.coffee

```diff
@@ -439,21 +441,26 @@ define (require) ->
         @jaxing = false
         @processCoachMath?()
 
-      # Update the hash fragment after the content has loaded
-      # to force the browser window to find the intended content
+      # Clear and replace the hash fragment after the content has loaded
+      # to force the browser window to find the intended content (as a side effect)
       jumpToHash = () =>
         if currentPage.get('loaded') and not @fragmentReloaded and window.location.hash
           @fragmentReloaded = true
           hash = window.location.hash
           window.location.hash = ''
           window.location.hash = hash
 
+          # scroll up by width of nav bar
+          # FIXME - make this dynamic
+          window.scrollBy(0, -145)
```

Comment:
Im working on implementing the updated header functionality stated by @philschatz in PR https://github.com/Connexions/webview/pull/1541